File: InterviewQuestions/Javascript Interview Programming Questions/task13.js
// Write a Javascript program to generate password of eight character

let charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
let password = "";
let required_length = 8

for (let i = 0; i < required_length; i++) {

    let randomIndex = Math.floor(Math.random() * charset.length);
    password = password + charset[randomIndex];

}

console.log(password);

File: InterviewQuestions/Python Interview Programming Questions/task13.py
# Write a Python program to generate password of eight character

import random

charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
password = ""
required_length = 8

for i in range(required_length):
    random_index = random.randint(0, len(charset) - 1)
    password = password + charset[random_index]

print(password)